app/src/main/java/tech/ula/ui/FilesystemEditFragment.kt

```diff
@@ -153,6 +180,72 @@ class FilesystemEditFragment : Fragment() {
         })
     }
 
+    private fun setupImportButton() {
+        import_button.setOnClickListener {
+            val filePickerIntent = Intent(Intent.ACTION_GET_CONTENT)
+            filePickerIntent.addCategory(Intent.CATEGORY_OPENABLE)
+            filePickerIntent.type = "application/*"
+            val fileChooser = Intent.createChooser(filePickerIntent, "Select Filesystem Backup file")
```

Comment:
If `Select Filesystem Backup file` can be a string resource, I think it should be. Capitalization could also be made uniform.